src/create-reducer.js

```diff
@@ -47,7 +47,8 @@ export default function createReducer(
         : stateAfterActions;
     if (state !== next) {
       computedProperties.forEach(({ parentPath, bindComputedProperty }) => {
-        bindComputedProperty(get(parentPath, next));
+        const prop = get(parentPath, next);
+        if (prop) bindComputedProperty(prop);
```

Comment:
Actually I have approved this PR already, but we should avoid implicit boolean conversion. I would change this to:

```typescript
if (prop != null) bindComputedProperty(prop);
```